espmonitor/src/main.rs

```diff
@@ -43,6 +43,9 @@ fn parse_args() -> Result<Option<AppArgs>, Box<dyn Error>> {
     if args.contains("-h") || args.contains("--help") {
         print_usage();
         Ok(None)
+    } else if args.contains("-v") || args.contains("--version") {
```

Comment:
GNU standard is generally a _capital_ `V` for the short option.  (Lowercase `v` is used for `--verbose`.)